### PR TITLE
4.2.0 packages

### PR DESCRIPTION
**Pull request by theopolis**
```
./tools/new_release.sh 4.2.0 ../osquery ../osquery-site
```

TODO: Sign and upload the MSI, update the hash.

### CODE REVIEW

**Comment by theopolis** [comment]
Heads up @muffins

**Comment by muffins** [comment]
I'll get the packages built and shipped upstream tonight/tomorrow when I have access to my keys

File: src/data/osquery_package_versions/4.2.0.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      {
+        "type": "Debian",
+        "package": "osquery_4.2.0_1.linux.amd64.deb",
+        "content": "76b4beb2c1eab0a9b3660ae54000d3e6db6c4845813e14ae0688316222620ac5",
+        "platform": "deb"
+      },
+      {
+        "type": "Windows",
+        "package": "osquery-4.2.0.msi",
+        "content": "0c5526ba75c617dfc3202573a184437d424a7b5af75d6a50e4e31dc4e820b5fd",
```
Comment (muffins) [inline src/data/osquery_package_versions/4.2.0.json]:
```
PS C:\Users\thor\Desktop\osquery_release_bins\4.2.0> ((Get-FileHash -Algorithm sha256 .\osquery-4.2.0.msi).Hash).ToLower()
2f6a3b1b263277e07e60739ea0c63c5dba5c74505a7d89448f9af707fce8c0ab
```

Comment (directionless) [inline src/data/osquery_package_versions/4.2.0.json]:
```suggestion
        "content": "2f6a3b1b263277e07e60739ea0c63c5dba5c74505a7d89448f9af707fce8c0ab",
```

**Pull request merged**